tests/core/rkyv.rs

```diff
@@ -5,16 +5,16 @@ use na::{
     Rotation3, Similarity3, SimilarityMatrix2, SimilarityMatrix3, Translation2, Translation3,
 };
 use rand;
-use rkyv::{Deserialize, Infallible};
 
 macro_rules! test_rkyv_same_type(
     ($($test: ident, $ty: ident);* $(;)*) => {$(
         #[test]
         fn $test() {
             let value: $ty<f32> = rand::random();
-			let bytes = rkyv::to_bytes::<_, 256>(&value).unwrap();
+			let bytes = rkyv::to_bytes::<rkyv::rancor::Error>(&value).unwrap();
 
-			let archived = rkyv::check_archived_root::<$ty<f32>>(&bytes[..]).unwrap();
+            let archived = rkyv::access::<<$ty<f32> as rkyv::Archive>::Archived, rkyv::rancor::Error>(&bytes)
+        .unwrap();
             // Compare archived and non-archived
 			assert_eq!(archived, &value);
 
```

Comment:
I imagine comparison between archived and original type is less automatic now, so it will have to be explicitly added for each type through `rkyv(compare(PartialEq))`

Reply:
Those must be derived manually when using `as`, otherwise `compare(PartialEq)`.